.github/workflows/linux.yml

```diff
@@ -267,7 +267,7 @@ jobs:
         run: |
           source ${OV_INSTALL_DIR}/setupvars.sh
           python -m pip install . --verbose --find-links ${OV_INSTALL_DIR}/wheels
-          python -m pytest -v ./tests/python_tests --ignore ./tests/python_tests/test_whisper_generate_api.py --ignore ./tests/python_tests/test_vlm_api.py -k "not test_set_chat_template"
+          python -m pytest -v ./tests/python_tests --ignore ./tests/python_tests/test_whisper_generate_api.py --ignore ./tests/python_tests/test_whisper_pipeline_static.py --ignore ./tests/python_tests/test_vlm_api.py -k "not test_set_chat_template"
```

Comment:
Why do we skip `test_whisper_pipeline_static` for linux?

Reply:
It's skipped for `genai_python_lib` job target, but added for `genai_python_lib_whisper` target (below).